pages/common/column.md

```diff
@@ -0,0 +1,19 @@
+# column
+
+> Format standard or file input into multiple columns, rows are filled before columns.
+
+- Output is formatted for a 10 columns wide display:
+
+`printf "header1 header2\nbar foo\n" | column -c {{10}}`
+
+- Specify characters to use to delimit columns for the -t option (i.e. "," for csv), default is whitespace:
```

Comment:
Would it be clear enough to simply say "Specify column delimiter character for the -t option"? The parenthesized note can remain.

Also, replace the last comma with a semicolon, to avoid a run-on sentence.